src/GitHub.App/Extensions/AkavacheExtensions.cs

```diff
@@ -252,11 +252,12 @@ public static class AkavacheExtensions
         /// <typeparam name="T"></typeparam>
         /// <param name="blobCache">The cache to retrieve the object from.</param>
         /// <param name="key">The key to look up the cache value with.</param>
-        /// <param name="item">The item to add to the database</param>
+        /// <param name="fetchFunc"></param>
```

Comment:
`The fetch function` - this is what the other params like like in the same file are documented with.